<p>This rule checks that the specified child tag does not appear as a direct child of the specified parent.</p>
<h2>Noncompliant Code Example</h2>
<p>Assuming a parent/child combination of <code>head</code>/<code>body</code>:</p>
<pre>
&lt;head&gt;
  ...
  &lt;body&gt;  &lt;!-- Noncompliant --&gt;
    ...
  &lt;/body&gt;
&lt;/head&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;head&gt;
  ...
&lt;/head&gt;
</pre>

