<p>Checker to find use of single quote where double quote is preferred.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;div id='header'&gt;&lt;/div&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;div id="header"&gt;&lt;/div&gt;
</pre>

