<p>Content that doesn't change or that doesn't change often should be included using a mechanism which won't try to interpret it. Specifically,
<code>&lt;%@ include file="..." %&gt;</code>, which includes the file in the JSP servlet translation phase (i.e. it happens once), should be used
instead of <code>&lt;jsp:include page="..." /&gt;</code>, which includes the page on the file, when the content is being served to the user.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;jsp:include page="header.jsp"&gt;  &lt;!-- Noncompliant --&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;%@ include file="header.jsp" %&gt;
</pre>

