<p>HTML &lt;table&gt; elements should not be used for layout purpose as it can confuse screen readers. It is recommended to use CSS instead.</p>
<p>This rule raises an issue on every <code>&lt;table&gt;</code> element containing a <code>role</code>&nbsp;attribute set to
<code>"presentation"</code> or <code>"none"</code>, which is how <a href="https://www.w3.org/WAI/tutorials/tables/tips/">W3C recommends</a> marks
layout tables.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;table role="presentation"&gt;
&lt;!-- ... --&gt;
&lt;/table&gt;
</pre>
<h2>See</h2>
<ul>
  <li> <a href="https://www.w3schools.com/html/html_layout.asp">HTML Layouts</a> </li>
</ul>

