<p>Even if all browsers are fault-tolerant, HTML tags should be closed to prevent any unexpected behavior.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;html&gt;
  &lt;head&gt;
    &lt;title&gt;Test Page    &lt;!-- Noncompliant; title not closed --&gt;
  &lt;!-- Noncompliant; head not closed --&gt;
  &lt;body&gt;
    &lt;em&gt;Emphasized Text  &lt;!-- Noncompliant; em not closed --&gt;
  &lt;!-- Noncompliant; body not closed --&gt;
&lt;/html&gt;
</pre>
<h2>Compliant Solution</h2>
<pre>
&lt;html&gt;
  &lt;head&gt;
    &lt;title&gt;Test Page&lt;/title&gt;
  &lt;/head&gt;
  &lt;body&gt;
    &lt;em&gt;Emphasized Text&lt;/em&gt;
  &lt;/body&gt;
&lt;/html&gt;
</pre>

