<p>This rule allows to make sure that all JSF Expressions are syntactically correct.</p>
<h2>Noncompliant Code Example</h2>
<pre>
&lt;?xml version="1.0" encoding="UTF-8"?&gt;
&lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml"
   xmlns:h="http://java.sun.com/jsf/html"&gt;
   &lt;h:body&gt;
     First name
     &lt;h:outputText value="#{user.firstName &amp;&amp; @@}"/&gt;   &lt;!-- Noncompliant --&gt;
   &lt;/h:body&gt;
&lt;/html&gt;
</pre>

